apollo-router-core/src/query_planner/mod.rs

```diff
@@ -4,10 +4,9 @@ mod selection;
 
 use crate::prelude::graphql::*;
 pub use caching_query_planner::*;
-use futures::lock::Mutex;
 use futures::prelude::*;
+use futures::stream::FuturesUnordered;
```

Comment:
:yarn: you already have it more or less in scope. Use `stream::FuturesUnordered` in your code.

```suggestion
```